api/docs.go

```diff
@@ -97,6 +97,30 @@ const docTemplate = `{
                 }
             }
         },
+        "/repositories/:uuid/rpms": {
+            "get": {
+                "description": "get repositories RPMS",
```

Comment:
```suggestion
                "description": "get repositories RPMS",
```
can you make all those RPMS into RPMs ? easier to read.

Reply:
thanks! updated